headers/assert_h.md

```diff
@@ -1,8 +1,63 @@
 # \<assert.h\>
 
-Reference: [\<assert.h\>](https://en.cppreference.com/w/c/assert)
+C Reference: [\<assert.h\>](https://en.cppreference.com/w/c/error)
 
-## Types
+Relevant Zig Documentation:
+  - [std.debug.assert](https://ziglang.org/documentation/master/std/#std;debug.assert)
 
-## Functions
+## [assert()](https://en.cppreference.com/w/c/error/assert)
 
+For the following C code, which will immediately abort the program if the condition is false. 
+
+```c
```

Comment:
I would add an `#include <assert.h>` here, just used to seeing it in manpages, same thing for importing std in the zig code example. A bit redundant but it allows for people to copy paste code

Reply:
True. Will do that.

Also I will add the following scripts for testing Zig & C code so that any future code reference will be always correct.

https://github.com/Sobeston/ziglearn/blob/master/do_tests.zig
https://github.com/Sobeston/ziglearn/blob/master/test-out.zig

Will also write a similar zig "script" for C code too in future too.